extensions/wikia/ArticleVideo/tests/ArticleVideoContextTest.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 class ArticleVideoContextTest extends WikiaBaseTest {
+	// FIXME add tests for new methods
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this "FIXME" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)